docker/1.14/Dockerfile.eia

```diff
@@ -85,17 +85,21 @@ COPY sagemaker /sagemaker
 WORKDIR /
 RUN mv /sagemaker/tensorflow/ /usr/local/lib/python3.*/site-packages/
 
-RUN wget https://amazonei-healthcheck.s3.amazonaws.com/v${HEALTH_CHECK_VERSION}/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -O /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
- && tar -xvf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -C /opt/ \
- && rm -rf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
- && chmod a+x /opt/ei_health_check/bin/health_check
+RUN wget https://amazonei-tools.s3.amazonaws.com/v${HEALTH_CHECK_VERSION}/ei_tools_${HEALTH_CHECK_VERSION}.tar.gz -O /opt/ei_tools_${HEALTH_CHECK_VERSION}.tar.gz \
+ && tar -xvf /opt/ei_tools_${HEALTH_CHECK_VERSION}.tar.gz -C /opt/ \
+ && rm -rf /opt/ei_tools_${HEALTH_CHECK_VERSION}.tar.gz \
+ && chmod a+x /opt/ei_tools/bin/health_check \
+ && mkdir -p /opt/ei_health_check/bin \
+ && ln -s /opt/ei_tools/bin/health_check /opt/ei_health_check/bin/health_check \
+ && ln -s /opt/ei_tools/lib /opt/ei_health_check/lib
 
 # Expose ports
 EXPOSE 8500 8501
 
-RUN wget https://amazonei-tensorflow.s3.amazonaws.com/tensorflow-serving/v1.14/ubuntu/archive/tensorflow-serving-${S3_TF_VERSION}-ubuntu-ei-${S3_TF_EI_VERSION}.tar.gz \
-            -O /tmp/tensorflow-serving-${S3_TF_VERSION}-ubuntu-ei-${S3_TF_EI_VERSION}.tar.gz \
- && cd /tmp \
+#RUN wget https://amazonei-tensorflow.s3.amazonaws.com/tensorflow-serving/v1.14/ubuntu/archive/tensorflow-serving-${S3_TF_VERSION}-ubuntu-ei-${S3_TF_EI_VERSION}.tar.gz \
```

Comment:
This needs another revision to uncomment and use the wget command once the binaries are available at the location.

Reply:
This new revision has fixed this